go/vt/vtgate/vindexes/lookup_internal.go

```diff
@@ -24,25 +24,29 @@ import (
 	"strconv"
 	"strings"
 
+	"vitess.io/vitess/go/vt/vterrors"
+
 	"vitess.io/vitess/go/sqltypes"
 
 	querypb "vitess.io/vitess/go/vt/proto/query"
 	vtgatepb "vitess.io/vitess/go/vt/proto/vtgate"
+	vtrpcpb "vitess.io/vitess/go/vt/proto/vtrpc"
 )
 
 // lookupInternal implements the functions for the Lookup vindexes.
 type lookupInternal struct {
-	Table         string   `json:"table"`
-	FromColumns   []string `json:"from_columns"`
-	To            string   `json:"to"`
-	Autocommit    bool     `json:"autocommit,omitempty"`
-	Upsert        bool     `json:"upsert,omitempty"`
-	IgnoreNulls   bool     `json:"ignore_nulls,omitempty"`
-	BatchLookup   bool     `json:"batch_lookup,omitempty"`
-	sel, ver, del string
+	Table                string   `json:"table"`
+	FromColumns          []string `json:"from_columns"`
+	To                   string   `json:"to"`
+	Autocommit           bool     `json:"autocommit,omitempty"`
+	MultiShardAutocommit bool     `json:"multi_shard_autocommit,omitempty"`
```

Comment:
Can't we use commonConfig here?

Reply:
that will make these property of a json object. which we should not do for backward compatibility.